CMakeLists.txt

```
@@ -556,6 +556,7 @@ if(VELOX_ENABLE_REMOTE_FUNCTIONS)
   # requires clients to explicitly install fizz and wangle.
```

```suggestion
  # requires clients to explicitly install fizz, wangle and xxHash.
```

We may want to add a link to https://github.com/facebookincubator/velox/issues/13153 for xxHash.